src/components/styles/Navbar.module.css

```diff
@@ -0,0 +1,69 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat&display=swap');
+
+.nav {
+  display: flex;
+  justify-content: space-between;
+  padding: 10px 8%;
+  font-family: 'Montserrat', sans-serif;
+  box-shadow: 1px 0 3px rgb(165, 164, 164);
+}
+
+.nav > div > ul {
+  list-style-type: none;
+  display: flex;
+  justify-content: space-between;
+  width: 170px;
+}
+
+.nav > div:nth-child(1) {
+  display: flex;
+  align-items: center;
+}
+
+.nav > div:nth-child(2) {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 40px;
+  height: 40px;
+  border-radius: 50%;
+  border: 1px solid #e8e8e8;
+}
+
+.nav > div:nth-child(1) h1 {
+  margin-right: 40px;
+  font-size: 1.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #0290ff;
+}
+
+.nav > div:nth-child(1) ul li * {
+  color: #121212;
+  text-decoration: none;
+  font-size: 0.813rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+}
+
+.icon {
+  color: #0290ff;
+}
+
+.activeLink {
+  opacity: 0.5;
+  font-family: Montserrat, sans-serif;
+  font-size: 0.813rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: #121212;
+}
```

Comment:
You did well with your project, kindly implement the following changes :

- Kindy use the right font used in the design given
- kindly use the right margin and padding in the navigation

Design given :
![image](https://github.com/Esmatullah12/book-store-react/assets/20537712/486426ce-009e-492e-8f87-bdad37ec8940)

Your design :
![image](https://github.com/Esmatullah12/book-store-react/assets/20537712/7ad4e251-30c4-4855-ab91-f3c2da78abb8)


- Please fix the margin and padding or general spacing issues you have in this part of your project too
![image](https://github.com/Esmatullah12/book-store-react/assets/20537712/51503bef-79b1-4e71-a319-acea60e136f6)

Kindly not that there should be a drop-down arrow in this part of your design as it is shown in the design given 😄 
![image](https://github.com/Esmatullah12/book-store-react/assets/20537712/f1a1a4b8-3b0f-48b9-a150-6c056b448885)